runner/ansible/roles/1.2.2/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Set Test ID
+  set_fact:
+    test_id: '1.2.2'
+
+- name: "{{ test_id }}.check"
+  shell: |
+   timeout=$(crm_attribute -t crm_config -G -n stonith-timeout --quiet)
+          if [[cibadmin -Q --xpath "//primitive[@type='fence_azure_arm']/@type" > /dev/null 2>&1 ]]; then
```

Comment:
Please, indent properly the script code, it is difficult to read otherwise
(try to do this in all the new shell code tasks)

Reply:
will do.